json-schema.gemspec

```diff
@@ -24,5 +24,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency "webmock"
   s.add_development_dependency "bundler"
 
-  s.add_runtime_dependency "addressable", '~> 2.3'
+  s.add_runtime_dependency "addressable", '~> 2.3', '>= 2.3.7'
```

Comment:
No, you're right (I thought that ~> 2.3.7 meant >= 2.3 && < 3.0 - but
apparently not)